src/sentry/search/events/builder/errors.py

```diff
@@ -40,6 +44,7 @@ def parse_query(self, query: str | None) -> ParsedTerms:
             self.params.user,
             list(filter(None, self.params.environments)),
             value_converters=value_converters,
+            allow_aggregate_filters=True,
```

Comment:
We're splitting queries to discover dataset to explicitly query errors or transactions datasets.

The `convert_query_values` doesn't allow aggregate filters but it's allowed in discover queries. Added this allow_aggregate_filters param to get around it so the ErrorsQueryBuilder is allowed to have aggregate filters and all other references to `convert_query_values` continue to not allow it. Want to make sure this is okay (@wedamija perhaps you'd know?)